suite-native/video-assets/scripts/generateVideos.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console */
+
+import fs from 'fs';
+import path from 'path';
+import prettier from 'prettier';
+import { exec } from 'child_process';
+
+const CONVERT_SCRIPT_PATH = './scripts/compress-videos.sh';
+const COMPRESSED_VIDEOS_PATH = '../assets/compressed';
+const VIDEOS_MAP_PATH = './src/videos.ts';
+
+/**
+ * Wraps child_process.exec into Promise to make it async.
+ */
+const asyncExec = (command: string) =>
```

Comment:
Why do you need it async? I would say you can even replace this whole function with simple `execSync` because we don't need any concurency here.

Reply:
fixed aa34c4297